### DISCUSSION TITLE
Avatar

### DISCUSSION THREAD

**Discussion by Febakke** [digdir/designsystemet#1817] [Components]
## Related information and insight
- Atlassian: https://atlassian.design/components/avatar/
- Ant: https://www.antforfigma.com/components/avatar

## Specification and requirements
- Should come in 3 sizes: `Small`, `medium`, `large`. 
- _Add specification and requirements_
- Should support initals, icons and an graphical elements (`picture`, `img`)

Issue: https://github.com/digdir/designsystemet/issues/1092

**Comment by mrosvik** [comment]
The component is created and published: https://next.storybook.designsystemet.no/?path=/docs/komponenter-avatar--docs
